content/en/apps/examples/direct-to-patient.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Direct-to-patient messaging workflows on CHT"
+linkTitle: "Direct-to-patient messaging workflows on CHT "
+weight:
+description: >
+  Reference for Direct-to-patient workflows with CHT and RapidPro
+relatedContent: >
```

Comment:
We should add an alias for `direct-to-client` since these are used somewhat interchangeably, and we will want to include direct-to-client term in this page as well.